.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -5,10 +5,11 @@ parameters:
 
 steps:
   - bash: |
-      if [[ -n ${SIG_GALLERY_NAME} && -n ${SIG_IMAGE_NAME_PREFIX} && -n ${SIG_IMAGE_VERSION} ]]; then m="sigMode"; sigImageName="${SIG_IMAGE_NAME_PREFIX}-${WINDOWS_SKU}"; else m="default"; fi && \
+      if [[ $(HYPERV_GENERATION) == "V2" ]]; then m="gen2Mode"; elif [[ -n ${SIG_GALLERY_NAME} && -n ${SIG_IMAGE_NAME_PREFIX} && -n ${SIG_IMAGE_VERSION} ]]; then m="sigMode"; sigImageName="${SIG_IMAGE_NAME_PREFIX}-${WINDOWS_SKU}"; else m="default"; fi && \
       echo "Set SIG_IMAGE_NAME to $sigImageName" && \
       echo "##vso[task.setvariable variable=SIG_IMAGE_NAME]$sigImageName" && \
       echo "Set build mode to $m" && \
+      echo "##vso[task.setvariable variable=SIG_IMAGE_NAME]$sigImageName" && \
```

Comment:
Why adding line 12 since we have set it in line 10?

Reply:
Yeah. Just noticed that. But we can define a $gen2sigImageName?

Reply:
Fixed.